### PR TITLE
Amend PDDA and add PDM project

### PR DESCRIPTION
**Pull request by scottatwell1**
### Jira link (if applicable)
Not applicable

### Change description ###
The PDDA project was located in the cnp-jenkins repo but has moved to sds-jenkins alongside PDM which is a sister project of PDDA.

### Checklist

- [x ] commit messages are meaningful and follow good commit message guidelines
- [x ] README and other documentation has been updated / added (if needed)
- [x ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: team-config.yml [hunk 1]
```diff
@@ -168,3 +168,21 @@ opal:
...
     application: opal
+pdda:
+  team: "PDDA"
+  namespace: "pdda"
+  azure_ad_group: DTS SDS PDDA
+  slack:
+    contact_channel: "#pdda"
+    build_notices_channel: "#pdda-builds"
+  tags:
+    application: pdda
```
Comment (timja) [inline team-config.yml:179]: was this assigned to you? if it wasn't can you expand the acronym at least so people not familiar with your project know what it is

File: team-config.yml [hunk 2]
```diff
@@ -168,3 +168,21 @@ opal:
...
+pdm:
+  team: "PDM"
+  namespace: "pdm"
+  azure_ad_group: DTS SDS PDM
+  slack:
+    contact_channel: "#pdm"
+    build_notices_channel: "#pdm-builds"
+  tags:
+    application: pdm
```
Comment (timja) [inline team-config.yml:188]: was this assigned to you? if it wasn't can you expand the acronym at least so people not familiar with your project know what it is

Comment (scottatwell1) [reply team-config.yml:188]: Hi. PDM is the Public Display Manager currently part of XHIBIT, but is being migrated as a standalone application alongside PDDA (Public Display Data Aggregator). PDM is used to manage the public displays in the crown courts.
PDDA will handle the retrieval of data from Common Platform to be display on the public displays; currently this is also handled by XHIBIT.

File: team-config.yml (hunk 1 above)
Comment (scottatwell1) [reply team-config.yml:179]: Hi. PDM is the Public Display Manager currently part of XHIBIT, but is being migrated as a standalone application alongside PDDA (Public Display Data Aggregator). PDM is used to manage the public displays in the crown courts.
PDDA will handle the retrieval of data from Common Platform to be display on the public displays; currently this is also handled by XHIBIT.

Comment (timja) [reply team-config.yml:179]:
```suggestion
    application: public-display-data-aggregator
```

File: team-config.yml (hunk 2 above)
Comment (timja) [reply team-config.yml:188]:
```suggestion
    application: public-display-manager
```

File: team-config.yml [hunk 3]
```diff
@@ -168,3 +168,21 @@ opal:
...
+    build_notices_channel: "#pdda-builds"
+  tags:
+    application: public-display-data-aggregator
+pdm:
+  team: "PDM"
+  namespace: "pdm"
+  azure_ad_group: DTS SDS PDM
+  slack:
+    contact_channel: "#pdm"
+    build_notices_channel: "#pdm-builds"
```
Comment (timja) [inline team-config.yml:186]: these slack channels don't seem to exist or are private and should not be

**Review by timja — CHANGES_REQUESTED** [review]

File: team-config.yml (hunk 3 above)
Comment (scottatwell1) [reply team-config.yml:186]: Apologies.
Those 2 slack channels have now been created and are public.

**Review by timja — APPROVED** [review]

**Pull request merged**